elastalert/elastalert.py

```diff
@@ -954,6 +959,7 @@ def init_rule(self, new_rule, new=True):
             elastalert_logger.warning('Error connecting to Elasticsearch for rule {}. '
                                       'The rule has been disabled.'.format(new_rule['name']))
             self.send_notification_email(exception=e, rule=new_rule)
+            raise e
```

Comment:
You only need to call `raise` (remove `e`)

http://www.ianbicking.org/blog/2007/09/re-raising-exceptions.html